Entities/Actor.h

```diff
@@ -1538,6 +1534,12 @@ ENTITYALLOCATION(Actor)
     // Timer for measuring interval between height checks
     Timer m_FallTimer;
 
+    /// <summary>
+    /// Gets a vector containing the script function names this class supports.
+    /// </summary>
+    /// <returns>A vector containing the script function names this class supports.</returns>
+    virtual const std::vector<std::string> GetSupportedScriptFunctionNames() override { auto functionNames = MOSRotating::GetSupportedScriptFunctionNames(); functionNames.insert(functionNames.end(), {"UpdateAI"}); return functionNames; }
```

Comment:
I think this is complicated enough to either be broken to multiple lines or moved to the cpp file.

Also, didn't we at one point agree that `auto` is cool but it obfuscates the code?
I think it can stay in lambdas but in other places it's better to write out the type.

Reply:
Replaced it with a preprocessor definition like a boss.